src/app/components/media/MediaComponent.js

```diff
@@ -23,6 +23,7 @@ import { getSuperAdminMask } from '../../helpers';
 import { units, Column } from '../../styles/js/shared';
 
 const StyledThreeColumnLayout = styled.div`
+  border-left: solid 2px var(--grayBorderMain);
```

Comment:
This was added temporarily, to keep the item page looking the same until we get around to those layout changes